app/components/billing-header/component.js

```diff
@@ -0,0 +1,22 @@
+import Ember from 'ember';
+
+const reads = Ember.computed.reads;
+
+export default Ember.Component.extend({
+  organization: null,
+  billingDetail: null,
+
+  isLoaded: Ember.computed.notEmpty('billingDetail'),
+  plan: reads('organization.plan'),
+  paymentMethodName: reads('billingDetail.paymentMethodName'),
+  paymentMethodDisplay: reads('billingDetail.paymentMethodDisplay'),
+  nextInvoiceDate: reads('billingDetail.nextInvoiceDate'),
+
+  init() {
+    this._super.apply(this, ...arguments);
```

Comment:
Can also:

```js
  init() {
    this._super(...arguments);
  }
```

It [does the right thing](http://babeljs.io/repl/#?experimental=false&evaluate=true&loose=false&spec=false&playground=true&code=Component.extend(%7B%0A%20%20init()%20%7B%0A%20%20%20%20this._super(...arguments)%3B%0A%20%20%7D%0A%7D)%3B).